amplitude_python_sdk/v2/ChartAnnotationsApiClient.py

```diff
@@ -0,0 +1,79 @@
+"""Client implementation for the Amplitude V2 Annotations API."""
+
+from typing import Optional, Dict, Any
+
+import requests
+
+from .models.charts import ChartAnnotationsV2
+from ..common.exceptions import AmplitudeAPIException
+from ..common.utils import return_or_raise
+
+
+class ChartAnnotationsApiClient:  # pylint: disable=too-few-public-methods
```

Comment:
I think making a separate client here is the wrong abstraction - we should keep chart annotations as a method under [`AmplitudeV2APIClient`](https://github.com/researchrabbit/amplitude-python-sdk/blob/73879e372eb807ff9124e97747d7b01d67fa83e2/amplitude_python_sdk/v2/client.py#L14) instead

Reply:
done